README.md

```diff
@@ -204,10 +207,12 @@ The next options are for the heuristic cost.
 
 `Heuristic Scale` can be used to force the algorithm to prefer exploring nodes that it thinks are closer to the goal.
 
-The last option, `Use Node Size Compensation`, is another optimization suggested by the paper. If enabled, both the traversal cost and the heuristic cost will be adjusted so that it's cheaper to go through big nodes than through small ones.
+The next option, `Use Node Size Compensation`, is another optimization suggested by the paper. If enabled, both the traversal cost and the heuristic cost will be adjusted so that it's cheaper to go through big nodes than through small ones.
 
 You can easily see the impact of all those options on the pathfinding computation by using the path finder test actor. See below for informations.
 
+The last option allows you to run a CatmullRom Algorithm on the points generated by the pathfinding to smooth it out. You can use the `Smoothing Subdivisions` to make control the smoothness.
```

Comment:
At the end of the sentence:
To control the smoothness
instead of
To make control the smoothness